build.sbt

```diff
@@ -135,23 +135,34 @@ lazy val docs = Project(
   // Make sure code generation is ran:
   .dependsOn(pluginTesterScala)
   .dependsOn(pluginTesterJava)
-  .enablePlugins(AkkaParadoxPlugin)
+  .enablePlugins(AkkaParadoxPlugin, ParadoxSitePlugin, PublishRsyncPlugin)
   .settings(
+    name := "Akka gRPC",
     skip in publish := true,
+    whitesourceIgnore := true,
+    previewPath := (Paradox / siteSubdirName).value,
+    Paradox / siteSubdirName := s"docs/akka-grpc/${if (isSnapshot.value) "snapshot" else version.value}",
     // Make sure code generation is ran before paradox:
     (Compile / paradox) := ((Compile / paradox) dependsOn (Compile / compile)).value,
-    Compile / paradoxGroups := Map(
+    paradoxGroups := Map(
       "Language" -> Seq("Java", "Scala"),
       "Buildtool" -> Seq("sbt", "Gradle", "Maven"),
     ),
     Compile / paradoxProperties ++= Map(
+      "akka.version" -> Dependencies.Versions.akka,
+      "akka-http.version" -> Dependencies.Versions.akkaHttp,
       "grpc.version" → Dependencies.Versions.grpc,
-      "project.url" -> "https://developer.lightbend.com/docs/akka-grpc/current/",
-      "canonical.base_url" -> "https://developer.lightbend.com/docs/akka-grpc/current/",
-      "akka-http.version" → Dependencies.Versions.akkaHttp,
-      "extref.akka-http.base_url" -> s"https://doc.akka.io/docs/akka-http/${Dependencies.Versions.akkaHttp}/%s",
+      "project.url" -> "https://doc.akka.io/docs/akka-grpc/current/",
+      "canonical.base_url" -> "https://doc.akka.io/docs/akka-grpc/current",
```

Comment:
These two are read by default from the `homepage` setting. So it might be better to set it like in Alpakka: https://github.com/akka/alpakka/blob/master/project/Common.scala#L20

Reply:
But here we use `akka.io` as `homepage`.

Reply:
Ahh, alright.